app/containers/HomePage/sagas.js

```diff
@@ -32,9 +33,7 @@ export function* getRepos() {
  * Watches for LOAD_REPOS action and calls handler
```

Comment:
Let's update this to say "Watches for `LOAD_REPOS` actions and calls `getRepos` when one comes in. By using `takeLatest` (and not `takeEvery`), only the result of the latest API call is applied." (or something like that)